internal/cli/atlas/metrics/disks_describe.go

```diff
@@ -70,6 +70,7 @@ func DisksDescribeBuilder() *cobra.Command {
 		PreRunE: func(cmd *cobra.Command, args []string) error {
 			return opts.PreRunE(
 				opts.ValidateProjectID,
+				opts.MetricsOpts.ValidatePeriodStartEnd,
```

Comment:
go has automatic delegation, I believe this can be simplified to 
```suggestion
				opts.ValidatePeriodStartEnd,
```

Same for other usages

Reply:
Ah, yes. I remember that... Go "flattens" nested structs - very nice feature. I'll change it...

Reply:
That has been addressed in the latest commit.